legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/modelToModelToRelational/milestoned_graphFetch_m2m2r.pure

```diff
@@ -0,0 +1,212 @@
+###Pure
+import meta::pure::mapping::*;
+import meta::pure::milestoning::*;
+import meta::pure::graphFetch::tests::m2m2r::milestoning::*;
+import meta::pure::graphFetch::execution::*;
+import meta::pure::alloy::connections::alloy::specification::*;
+import meta::pure::alloy::connections::alloy::authentication::*;
+import meta::relational::runtime::*;
+import meta::pure::runtime::*;
+
+function <<test.Test, test.AlloyOnly>> meta::pure::graphFetch::tests::m2m2r::milestoning::toMilestonedTargetProperty::testToMilestonedTargetProperty_hardcodedDate():Boolean[1]
+{
+  let mapping =meta::relational::tests::m2m2r::milestoning::toMilestonedTargetProperty::TargetToModelMappingWithMilestonedComplexProperty;
+  let runtime=  getModelChainRuntime($mapping);
+
+   let query={|meta::relational::tests::milestoning::TargetProduct1.all(%2023-10-15T00:00:00)->graphFetch(
+      #{
+        meta::relational::tests::milestoning::TargetProduct1{
+          id,
+          name,
+          synonyms(%2020-10-15T00:00:00)
+          {synonym}
+        }
+      }#
+    )->serialize(
+      #{
+        meta::relational::tests::milestoning::TargetProduct1{
+          id,
+          name,
+          synonyms(%2020-10-15T00:00:00)
+          {synonym}
+        }
+      }#
+    )
+    ->meta::pure::mapping::from($mapping,$runtime)
+    };
+   let result = meta::legend::executeLegendQuery($query, [], ^meta::pure::runtime::ExecutionContext(), meta::relational::extension::relationalExtensions());
+   assertJsonStringsEqual('{"builder":{"_type":"json"},"values":[{"id":2,"name":"ProductName2","synonyms(2020-10-15T00:00:00+0000)":[{"synonym":"GS-Mod-S1"},{"synonym":"GS-Mod-S3"}]},{"id":3,"name":"ProductName3","synonyms(2020-10-15T00:00:00+0000)":[{"synonym":"GS-Mod-S3"}]}]}', $result);
+}
+
+function <<test.Test, test.AlloyOnly, test.ToFix>>  meta::pure::graphFetch::tests::m2m2r::milestoning::toMilestonedTargetProperty::testToMilestonedTargetProperty_variableDate():Boolean[1]
```

Comment:
this test needs fixing of date variable handling in relational graph fetch . will fix in upcoming PR with some other changes